data-share/data-share-service/src/main/java/io/mosip/datashare/service/impl/DataShareServiceImpl.java

```diff
@@ -119,6 +120,13 @@ public class DataShareServiceImpl implements DataShareService {
 	@Value("${mosip.data.share.protocol}")
 	private String httpProtocol;
 
+	/** Defines whether static data share policy needs to be used for sharing the data*/
+	@Value("${mosip.data.share.standalone.mode.enabled:false}")
+	private boolean standaloneModeEnabled;

```

Comment:
Can we add a section in the readme file, about this new mode and how to activate this using the new property file in the mosip config server ? 
We can create a task on this, we can take this after giving the build to the QA, since it is just a readme file change.

Reply:
Sure @vishwa-vyom will create a task for the same.